src/MovieCard/MovieCard.test.js

```diff
@@ -2,37 +2,25 @@ import React from 'react';
 import { fireEvent, screen, render } from '@testing-library/react';
 import MovieCard from './MovieCard.js';
 import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
 
 describe('MovieCard', () => {
   it('MovieCard should render correctly', () => {
-    render(<MovieCard 
-      title={ 'Armagedon' }
-      posterPath={ 'https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg' }
-      id={ 694919 }
-      key={ 694919 }
-      movieEventClick={ jest.fn() }
-     />
-    )
+    render(
+      <MemoryRouter>
+        <MovieCard
+          title={'Armagedon'}
+          posterPath={
+            'https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg'
+          }
+          id={694919}
+          key={694919}
+          movieEventClick={jest.fn()}
+        />
+      </MemoryRouter>
+    );
 
     expect(screen.getByText('Armagedon')).toBeInTheDocument();
     expect(screen.getByAltText('Armagedon')).toBeInTheDocument();
-  })
-
-  it('should fire an closing button event', () => {
-    const mockMovieClickEvent = jest.fn()
-    render(<MovieCard 
-      title={ 'Armagedon' }
-      posterPath={ 'https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg' }
-      id={ 694919 }
-      key={ 694919 }
-      movieEventClick={ mockMovieClickEvent }
-     />
-    )
```

Comment:
Movie preview exit function on click event no longer called, good to remove its testing